docker/init.sls

```diff
@@ -96,6 +96,22 @@ docker-config:
     - mode: 644
     - user: root
 
+{% if salt['grains.has_value']('systemd') %}
+docker-unit-drop-in:
```

Comment:
The grain exists on `systemd` nodes and does not exist on non-`systemd` nodes.  `grains.has_value` returns True if the grain exists and has a value.